icu4c/source/test/testdata/ucptrie/free-blocks.16.toml

```diff
@@ -0,0 +1,102 @@
+[code_point_trie.struct]
+name = "free-blocks.16"
+index = [
+	0,0x40,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```

Comment:
Is the indent required?
(Waste of space...)

Reply:
Discussed.